### ISSUE TITLE
Setting stty options

### ISSUE THREAD

**Issue by webern** [gnome-terminator/terminator#564]
**Is your feature request related to a problem? Please describe.**

TL;DR

Is it possible to set `stty` options automatically when a new terminal is created? For example, whenever creating a new terminal I want something like `stty intr ^j` to happen.

Backstory:

I would like to copy and paste with control-c and control-v. I'm switching to Linux from macOS and I cannot make the adjustment to using control-shift-c and control-shift-v only when in the terminal. It's a comical nightmare how frequently control-shift-c spills over to other applications.

I have partially solved the problem with

```
[global_config]
[keybindings]
  copy = <Primary>c
  paste = <Primary>v
```

However this isn't the full solution because control-c still sends the interrupt signal. It seems like, so long as I have something highlighted, maybe the interrupt is not sent. But to be on the safe side I was thinking I would always map the interrupt to a something else, e.g. `stty intr ^j`. It seems like I would need to do this for each new session.

I may be conflating the role of the terminal emulator with the shell itself here, since it occurs to me that I would want this remapping so be consistent even after `ssh`-ing.

**Comment by mattrose** [comment]
This is definitely something that can (and should) be solved outside of terminator.  The easiest way is to just do the stty setting in your .bashrc (or .zshrc) file.
